Cargo.toml

```diff
@@ -41,7 +42,7 @@ serde_derive = "1.0"
 
 [features]
 default = ["std"]
-std = []
+std = ["borsh?/std"]
```

Comment:
It's better with `?`, but I'm still wondering if we need this at all -- especially since borsh uses `hashbrown` and `std` as mutually exclusive features, so we shouldn't force that choice as a library.